src/main/resources/answer_definitions_schema.json

```diff
@@ -0,0 +1,156 @@
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "ethnicity": {
+          "id": "#ethnicity",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, null ]
+        },
+        "ethnicity_other": {
+          "id": "#ethnicity_other",
+          "type": [ "string", "null" ]
+        },
+        "religion": {
+          "id": "#religion",
+          "type": [ "integer", "null" ],
+          "enum": [ 1, 2, 3, 4, 5, 6, 7, 8, null ]
+        },
+        "religion_other": {
+          "id": "#religion_other",
+          "type": [ "string", "null" ]
+        },
+        "disability_conditions": {
+          "id": "#disability_conditions",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_impact": {
+          "id": "#disability_impact",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, 3, null ]
+        },
+        "disability_vision": {
+          "id": "#disability_vision",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_hearing": {
+          "id": "#disability_hearing",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_mobility": {
+          "id": "#disability_mobility",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_dexterity": {
+          "id": "#disability_dexterity",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_learning": {
+          "id": "#disability_learning",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_memory": {
+          "id": "#disability_memory",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_health": {
+          "id": "#disability_health",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_stamina": {
+          "id": "#disability_stamina",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_social": {
+          "id": "#disability_social",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "disability_condition_other": {
+          "id": "#disability_condition_other",
+          "type": [ "string", "null" ]
+        },
+        "disability_none": {
+          "id": "#disability_none",
+          "type": [ "integer", "null" ],
+          "enum": [ 0, 1, 2, null ]
+        },
+        "pregnancy": {
+          "id": "#pregnancy",
+          "type": ["integer", "null"],
+          "enum": [ 0, 1, 2, null ]
+        }
+      },
+      "additionalProperties": false
```

Comment:
Maybe put json and xml files in there own folders as the resouces can be a bit messy as the project gets more complex.